src/pages/Home/components/AnimatedSVG.jsx

```diff
@@ -0,0 +1,637 @@
+// AnimatedSVG.js
+import React from "react";
+import { gsap } from "gsap";
+import { useGSAP } from "@gsap/react";
+import { ScrollTrigger } from "gsap/ScrollTrigger";
+import { MotionPathPlugin } from "gsap/MotionPathPlugin";
+import { EasePack } from "gsap/EasePack";
+import { CustomEase } from "gsap/CustomEase";
+import "../styles/svgAnimation.css";
+gsap.registerPlugin(EasePack, CustomEase, ScrollTrigger, MotionPathPlugin);
+
+gsap.registerPlugin(ScrollTrigger, MotionPathPlugin);
+
+const AnimatedSVG = () => {
```

Comment:
at certain page sizes, this animated SVG breaks layout
<img width="1215" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/ab698ca8-0e57-4bf4-8a55-302c5e86a967">

Reply:
I'm not getting this fully, can you please explain more ?

Reply:
> at certain page sizes,

e.g. ipad pro `1024px` wide, page loads like this (with overflows)

<img width="1110" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/65fcfdb8-2e84-49fe-bd81-5f1387169ae8">

Reply:
Actually this is difference between the breakout considered for the better.giving website and our landing page design, so there exists conflicting situation in this 1000px range

Reply:
@Navansh We can address those break point discrepancies later internally. If it look OK for now at a general desktop & mobile size, we can let this slide.  
cc: @ap-justin